ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -151,29 +151,6 @@ def _get_dtype_and_square_matrix(draw):
     return dtype, mat
 
 
-# bmm helper function
-@st.composite
```

Comment:
hey! thanks for contributing! please don't remove anything from the file. you should only add the test for the function you're working on, thanks!

Reply:
I made a git pull request . I think that's why  some line of code got deleted